lib/internal/fs/glob.js

```diff
@@ -0,0 +1,202 @@
+'use strict';
+const { lstatSync, readdirSync } = require('fs');
+const { join, resolve } = require('path');
+
+const {
+  kEmptyObject,
+} = require('internal/util');
+const { isRegExp } = require('internal/util/types');
+const {
+  validateFunction,
+  validateObject,
+} = require('internal/validators');
+
+const {
+  ArrayPrototypeForEach,
+  ArrayPrototypeMap,
+  ArrayPrototypeFlatMap,
+  ArrayPrototypePop,
+  ArrayPrototypePush,
+  SafeMap,
+  SafeSet,
+} = primordials;
+
+let minimatch;
+function lazyMinimatch() {
+  minimatch ??= require('internal/deps/minimatch/index');
```

Comment:
```suggestion
  minimatch ??= require('internal/deps/minimatch');
```
? it seems like this should either be the shortest path or the full one with extension, or am i misunderstanding how internal things work?

Reply:
It's not real "require" it's the built in require that just bundles all the JS with the Node.js executable

Reply:
ah ok, if this is fine then, please resolve